docs/specs/code/EXPORT_TRANSACTION_HISTORY_TO_FILE.md

```diff
@@ -0,0 +1,67 @@
+## Abstract
+
+Users want to export their transaction history into an Excel document.
+
+Default format gotta be XLSX, but optionally we might also support CSV, cuz it's more universal and might be imported into Excel as well as lots of other programs.
+
+Users should be able to chose the time period to export transaction for, like day from and day to. There also might be a list of default periods, e.g.: "this month", "last 30 days", "this quarter", "this year", "all". Default period selected might be "this month".
+
+## UI
+
+I suggest adding a new "Export to File" icon at the top-right of the transactions list. Something like this (general idea):
+
+![image](https://user-images.githubusercontent.com/5585355/50343034-8e94cc80-0536-11e9-9058-1af18be67f8a.png)
+
+When user clicks "Export to file" - new dialog window opens up:
+
+![image](https://user-images.githubusercontent.com/5585355/50346357-31534800-0543-11e9-949a-873bcae9b854.png)
+
+Dialog contains:
+1. Two date selectors: "from day", "to day"
+2. Dropdown-selector "Select period"
+3. Dropdown-selector: "File format"
+4. Button "Export"
+
+Comments:
+1. It might be an option for date-selectors to be locked/disabled by default, and only display the period selected in the dropdown, unless user selects option "Custom" - at which point selectors become enabled. But when user selects any other option - selectors become disabled again.
+2. If we only gonna support XSLT for now - the dropdown selector for format might be either locked/disabled on the only default option, or replaced with just a label.
```

Comment:
Arbitrary, just because we initially planned to do only XSLT and then CSV was added kinda as a backup. The same logic is applied here, like: people that know they can import CSV anywhere - will be able to select CSV. But at least people that don't know about it will get the most common and popular format by default.